package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.2.5",
     "debug": "^4.1.1",
+    "os": "^0.1.1",
```

Comment:
o_O 
this should be already integrated in NodeJS 
look: https://nodejs.org/api/os.html#os_os_hostname
@Eirenliel

Reply:
You are right, it was mistake, I removed the dependency.